autotools-wrappers/PKGBUILD

```diff
@@ -0,0 +1,54 @@
+# Maintainer: Christoph Reiter <reiter.christoph@gmail.com>
+
+pkgbase=autotools-wrappers
+pkgname=('automake-wrapper' 'autoconf-wrapper' )
+pkgver=20221207
+pkgrel=1
+pkgdesc="Wrapper scripts for autoconf & automake commands"
```

Comment:
Redundant pkgdesc?

Reply:
hm, true, thanks. I always forget that pkgbase doesn't really exist in the repo